pkg/config/config.go

```diff
@@ -893,11 +897,12 @@ func DefaultConfig() (*Config, error) {
 			EnableCriuSupport:           true,
 		},
 		ImageConfig: ImageConfig{
-			DefaultTransport:   "docker://",
-			PauseImage:         DefaultPauseImage,
-			PauseCommand:       "/pause",
-			ImageVolumes:       ImageVolumesMkdir,
-			SignaturePolicyDir: "/etc/crio/policies",
+			DefaultTransport:              "docker://",
+			PauseImage:                    DefaultPauseImage,
+			PauseCommand:                  "/pause",
+			ImageVolumes:                  ImageVolumesMkdir,
+			SignaturePolicyDir:            "/etc/crio/policies",
+			AutomaticallyReloadRegistries: false,
```

Comment:
Don't have to explicitly initialize it to `false` as this is a default value for bool (i.e. it's initialized to false if you omit explicit initialization).